src/components/dialog/index.tsx

```diff
@@ -247,7 +247,7 @@ class Dialog extends Component<DialogProps, DialogState> {
 
   render = () => {
     const {modalVisibility} = this.state;
-    const {testID, supportedOrientations, accessibilityLabel, ignoreBackgroundPress} = this.props;
+    const {testID, supportedOrientations, accessibilityLabel, ignoreBackgroundPress, ...others} = this.props;
```

Comment:
This should be done with `modalProps`:

```
  /**
   * Additional props for the modal.
   */
  modalProps?: ModalProps;
```

Reply:
@M-i-k-e-l 
We are currently declaring that Dialog props are extending Modal props. (code wise, we don't do it)
If you want to go with adding `modalProps` which I agree is safer, we should fix the component typings as well

Reply:
We have a `Pick` from `ModalPropsIOS`, or did I miss something? Generally we should remove that if we're adding the suggested prop, but I don't think it's worth the migration since we're going to move to the new `Dialog` soonish.

Reply:
Fixed :)